src/Expression.php

```diff
@@ -468,42 +468,27 @@ function ($matches) use (&$nameless_count) {
      */
     public function getDebugQuery($html = null)
```

Comment:
better no HTML here, `SqlFormatter` is sudgested anyway.

and should be fixed in data/ui

Reply:
You mean to deprecate the `$html` argument?

Reply:
yes - in this PR is not for rename ;-)